admin_manual/configuration_server/theming.rst

```diff
@@ -2,60 +2,62 @@
 Theming
 =======
 
-Nextcloud has two ways of theming. There is a theming app that provides a web
-UI to set up theming. The second option allows to overwrite most of the files
-but a customized theme needs to be created and maintained. For more information on
-this check out the `developers documentation <../../developer_manual/core/theming.html>`_.
+With our theming feature you are able to customize the look and feel of your
+Nextcloud instance according to the corporate design of your organization by
+replacing Nextcloud logo and color with your own assets.
 
-We recommend to use the theming app, because the Nextcloud team maintains this
-and verifies that all adjustments to the server work with the theming. It
-covers also most use cases like replacing the name, logos and colors.
+.. note:: Nextcloud also provides an more advanced but unmaintained theming which
```

Comment:
Should we just call it deprecated? If it is unmaintained, we probably don't need to mention it. Any thoughts @nextcloud/theming

Reply:
Removed the whole section. ✅ Documentation is still live in the developer-docs for now.
Should be enough. You are right. At some point, this might get removed anyways - since your CustomCSS-app is replacing this quit well 😇